phpcs.xml

```diff
@@ -1,35 +1,9 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework coding standard">
-    <description>Zend Framework coding standard</description>
+    <rule ref="./vendor/zendframework/zend-coding-standard/ruleset.xml"/>
 
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
     <arg name="extensions" value="php,dist,phtml"/>
 
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
-    <rule ref="PSR1.Files.SideEffects">
-        <exclude-pattern>public/index.php</exclude-pattern>
-        <exclude-pattern>bin/install.php</exclude-pattern>
-    </rule>
-    <rule ref="Generic.Files.LineLength.TooLong">
-        <exclude-pattern>config/autoload/global.php</exclude-pattern>
-        <exclude-pattern>bin/install.php</exclude-pattern>
-    </rule>
```

Comment:
A number of these rules provided exclusions for configuration and script files; why were those removed?

Reply:
I've done it because of moving to zend-coding-standard. I've reverted the exclusion in 927fc260a2a4558c690f1be352eddfd0f8fd0260
We are not running anywhere cs-check - there is no any Travis CI config with the repository...